### ISSUE TITLE
GUI Questions

### ISSUE THREAD

**Issue by Metallicow** [loot/loot#481]
This isn't an issue but just a questions 'thread' dump.
I primarily use wxPython for GUI development overall; often for everything and othertimes when I am working on Wrye Bash.

The first question is "What GUI Library are you using?"
My interest lies with the "tabs" dc/rendering widget.
When clicking on a "tab", it creates a gradient flowing omni-circle effect.
Is this GUI effect coming from the library you are using or is it something you specially coded yourself?

Thanks wrinkly.
- Metalio Bovinus

**Comment by WrinklyNinja** [comment]
I'm using [Polymer](https://www.polymer-project.org/1.0/) for LOOT's GUI. The tabs are from its [paper-tabs](https://elements.polymer-project.org/elements/paper-tabs) element, the ripple effect is not something I did myself.

I don't think I'd recommend building an HTML UI for a desktop application though, it's resource-intensive and the freedom it gives is a double-edged sword. I'd previously only used wxWidgets, which is OK but nothing to write home about: I'll probably try out Qt for my next GUI project, whatever that may be.

**Comment by Metallicow** [comment]
Ok, haven't run across that one yet. Is there an option to build LOOT with a native looking interface with Polymer or is it all custom drawing with that kit?

We used Qt for the Embroidermodder2 project and I'm still working on the python port with the bindings.
As far as Qt vs wx, I still prefer wx. The Qt Docking library is a bit smoother running visually, than wxPythons agw aui, but not quite as easily extendable in my opinion. wx also has slated a C/C++ rewrite of AUI in the next or future, so maybe all the extra python fancyness can be incorporated, since the C/C++ side orig coding/fixing guy basically fell off the planet. Kivy looks pretty neat also and is python gui kit for mobile touchscreens. JavaFX demo had a few cool slinky carousel widgets, but lacked most of the other custom stuff I have already wrote in wxPy piece by piece over the years.
